src/components/WwwFrame/PromotionalBanner/Banners/PromoCreditBanner.vue

```diff
@@ -22,7 +40,6 @@
 		</template>
 		<template
 			v-else
-			v-kv-track-event="['TopNav','click-Promo','Lending Reward Banner']"
```

Comment:
You can't have a directive on a temporary template element that vue is going to strip out anyway